plugins/rhs_plugin/index.js

```diff
@@ -14,9 +18,14 @@ function mapStateToProps(state) {
     const pluginComponent = rhsPlugins.find((element) => element.id === pluggableId);
     const pluginTitle = pluginComponent ? pluginComponent.title : '';
 
+    const isBoards = pluginComponent.pluginId === suitePluginIds.focalboard || pluginComponent.pluginId === suitePluginIds.boards;
```

Comment:
@hmhealey should we keep doing the validation for boards this way?

Reply:
Yes we still needs to check for both focalboard and boards